Mathlib/LinearAlgebra/Matrix/Determinant/TotallyUnimodular.lean

```diff
@@ -79,36 +78,82 @@ lemma transpose_isTotallyUnimodular_iff (A : Matrix m n R) :
     Aᵀ.IsTotallyUnimodular ↔ A.IsTotallyUnimodular := by
   constructor <;> apply IsTotallyUnimodular.transpose
 
-lemma mapEquiv_isTotallyUnimodular {X' Y' : Type*} (A : Matrix m n R) (eX : X' ≃ m) (eY : Y' ≃ n) :
-    IsTotallyUnimodular ((A · ∘ eY) ∘ eX) ↔ A.IsTotallyUnimodular := by
+lemma IsTotallyUnimodular.reindex {A : Matrix m n R} (eX : m ≃ m') (eY : n ≃ n')
+    (hA : A.IsTotallyUnimodular) :
+    IsTotallyUnimodular (A.reindex eX eY) :=
+  hA.submatrix _ _
+
+lemma reindex_isTotallyUnimodular (A : Matrix m n R) (em : m ≃ m') (en : n ≃ n') :
+    IsTotallyUnimodular (A.reindex em en) ↔ A.IsTotallyUnimodular := by
   rw [isTotallyUnimodular_iff, isTotallyUnimodular_iff]
```

Comment:
Here you should be able to golf using `Matrix.reindex_reindex` or whatever it's called.